src/sql/workbench/contrib/notebook/browser/outputs/notebookMarkdown.ts

```diff
@@ -240,7 +241,16 @@ export class NotebookMarkdownRenderer {
 		} else if (href.charAt(0) === '/') {
 			return base.replace(/(:\/*[^/]*)[\s\S]*/, '$1') + href;
 		} else if (href.slice(0, 2) === '..') {
-			return path.join(base, href);
+			if (isWindows) {
```

Comment:
Here you're checking if it's windows before doing the replace, but the other place you're just doing it for everything. We should be consistent with whether we check or not